x-pack/test_serverless/functional/test_suites/observability/navigation.ts

```diff
@@ -33,7 +33,6 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
       await svlCommonNavigation.breadcrumbs.expectExists();
 
       // check side nav links
-      await svlCommonNavigation.sidenav.expectSectionOpen('observability_project_nav');
```

Comment:
Wanted to double-check why these checks are no longer needed. Was the behavior incorrect and was it fixed by this PR?

Reply:
Yes this PR correctly sets the `isCollapsible: false` on the underlying EUI component. At that point, the section is no more an accordion, and thus does not have any more "arrow" (which `expectSectionOpen` relies on to make its check).